internal/turso/databases.go

```diff
@@ -207,6 +209,34 @@ func (d *DatabasesClient) Update(database string) error {
 	return nil
 }
 
+type Body struct {
+	Org string `json:"org"`
+}
+
+func (d *DatabasesClient) Transfer(database, org string) error {
+	url := d.URL(fmt.Sprintf("/%s/transfer", database))
+	body, err := json.Marshal(Body{Org: org})
+	bodyReader := bytes.NewReader(body)
+	if err != nil {
+		return fmt.Errorf("could not serialize request body: %w", err)
+	}
+	r, err := d.client.Post(url, bodyReader)
+	if err != nil {
+		return fmt.Errorf("failed to transfer database")
+	}
+	defer r.Body.Close()
+	if r.StatusCode == http.StatusForbidden {
+		err = parseResponseError(r)
+		return fmt.Errorf("%v", err)
+	}
+
+	if r.StatusCode != http.StatusOK {
+		return fmt.Errorf("failed to transfer database to org %s", org)
```

Comment:
What happens when the transfer succeeds but the update fails?

Reply:
The inspect and plan data will be dummy. I will add a comment asking for updating it

Reply:
Let's pretend it is atomic on the CLI.
Let's open an API issue to make it atomic.